src/main/java/org/folio/rest/impl/UserGroupAPI.java

```diff
@@ -169,7 +164,7 @@ public void getGroupsByGroupId(String groupId, String lang, Map<String, String>
         System.out.println("sending... getGroupsByGroupId");
 
         Criterion c = new Criterion(
-          new Criteria().addField(ID_FIELD_NAME).setJSONB(false).setOperation("=").setValue("'"+groupId+"'"));
+          new Criteria().addField(ID_FIELD_NAME).setJSONB(false).setOperation("=").setVal(groupId));
```

Comment:
Can you replace the complete method by a PgUtil.getById invokation so that this duplicated code gets removed?